astropy/vo/client/conesearch.py

```diff
@@ -81,7 +83,7 @@ def conesearch(center, radius, verb=1, **kwargs):
     Parameters
     ----------
     center : `~astropy.coordinates.SkyCoord`, `~astropy.coordinates.BaseCoordinateFrame`, or sequence of length 2
-        Position of the center of the cone to search.  It may be specified as an
+        Position of the center of the cone to search. It may be specified as an
```

Comment:
I deleted it mainly to stop my Emacs from complaining that it exceeded 80 characters.

But on a more personal note, using double whitespace after a period does anger me. I can't explain it.